### ISSUE TITLE
Opened log-file after restarted `VSCode`

### ISSUE THREAD

**Issue by vovs03** [LuaLS/vscode-lua#85]
### Which OS are you using?

MacOS

### Expected Behaviour

I has restarted `VSCode`. And after that, the log-file was opened. Log added at next field of this issue

### Actual Behaviour

...

I also see the following error:

```
[2023-03-03 11:20:02] |   ERROR   |         Git        | fatal: not a git repository (or any of the parent directories): .git

```]]

### Reproduction steps

1. Go to '...'
2. Click '...'
3. See error '...'


### Additional Notes

_No response_

### Log

#### STARTUP ####
[2023-03-03 11:20:01] |   DEBUG   |     Filesystem     | Created directory at "/Users/husky/Library/Application Support/Code/User/globalStorage/sumneko.lua/addonManager"
[2023-03-03 11:20:02] |   DEBUG   |         Git        | Cloned LLS-Addons
[2023-03-03 11:20:02] |   ERROR   |         Git        | fatal: not a git repository (or any of the parent directories): .git

**Comment by carsakiller** [comment]
Hello 👋

This issue should be fixed in `3.6.16`, there was an issue with file paths on UNIX machines.

Please confirm if this is no longer occurring.

**Comment by carsakiller** [comment]
Please follow this issue in https://github.com/LuaLS/vscode-lua/issues/81
